packages/components/src/Badge/Badge.component.js

```diff
@@ -28,12 +28,25 @@ function renderDeleteIcon(onClick, id, disabled, t) {
 	return null;
 }
 
-function Badge({ id, label, category, onDelete, onSelect, selected, disabled, t, style }) {
+function Badge({
+	id,
+	label,
+	category,
+	onDelete,
+	onSelect,
+	selected,
+	disabled,
+	t,
+	style,
+	className,
+}) {
 	const containerClasses = classNames(
-		'tc-badge', theme['tc-badge'],
+		'tc-badge',
```

Comment:
there is not value tested to add the tc-badge global and local class is this wanted ?

Reply:
That's why I've added the className to be able to specify it from the outside.